src/public/CowSwap.json

```diff
@@ -891,63 +891,63 @@
     },
     {
       "address": "0xb0C5f3100A4d9d9532a4CfD68c55F1AE8da987Eb",
       "chainId": 100,
       "name": "DAOhaus Token from Ethereum",
       "symbol": "HAUS",
       "decimals": 18,
-      "logoURI": "https://assets.coingecko.com/coins/images/14551/small/jN3kkqke_400x400.png"
+      "logoURI": "https://assets.coingecko.com/coins/images/14551/small/jn3kkqke_400x400.png"
     },
     {
       "address": "0xb7D311E2Eb55F2f68a9440da38e7989210b9A05e",
       "chainId": 100,
       "name": "Stake on Gnosis Chain",
       "symbol": "STAKE",
       "decimals": 18,
-      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x0Ae055097C6d159879521C384F1D2123D1f195e6/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x0ae055097c6d159879521c384f1d2123d1f195e6/logo.png"
     },
     {
       "address": "0xC45b3C1c24d5F54E7a2cF288ac668c74Dd507a84",
       "chainId": 100,
       "name": "Symmetric",
       "symbol": "SYMM",
       "decimals": 18,
-      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x57dB3FfCa78dBbE0eFa0EC745D55f62aa0Cbd345/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x57db3ffca78dbbe0efa0ec745d55f62aa0cbd345/logo.png"
     },
     {
       "symbol": "EURe",
       "name": "Monerium EUR emoney",
       "address": "0xcB444e90D8198415266c6a2724b7900fb12FC56E",
       "decimals": 18,
       "chainId": 100,
-      "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/100/0xcB444e90D8198415266c6a2724b7900fb12FC56E/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/100/0xcb444e90d8198415266c6a2724b7900fb12fc56e/logo.png"
     },
     {
       "address": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
       "chainId": 100,
       "name": "USD//C on Gnosis Chain",
       "symbol": "USDC",
       "decimals": 6,
-      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48/logo.png"
     },
     {
       "address": "0xE2e73A1c69ecF83F464EFCE6A5be353a37cA09b2",
       "chainId": 100,
       "name": "ChainLink Token on Gnosis Chain",
       "symbol": "LINK",
       "decimals": 18,
-      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x514910771AF9Ca656af840dff83E8264EcF986CA/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x514910771af9ca656af840dff83e8264ecf986ca/logo.png"
```

Comment:
Not all of them can be auto-lower cased, unfortunately.
This one for example results in a 404.

The best approach would be to lower case the `logoURI` ONLY if it's pointing to this repo.

Reply:
Damn, really, thanks!

Reply:
FIxed